internal-client/src/main/java/org/apache/uniffle/client/request/RssGetInMemoryShuffleDataRequest.java

```diff
@@ -65,4 +83,19 @@ public int getReadBufferSize() {
   public Roaring64NavigableMap getExpectedTaskIds() {
     return expectedTaskIds;
   }
+
+  @Override
```

Comment:
The reason of make RetryRequest as an abstract class is to extra the following retryMax params and so on into the RetryRequest class, rather than scattering everywhere

Reply:
Done.